app/models/ann_data_ingest_parameters.rb

```diff
@@ -0,0 +1,66 @@
+# handles launching ingest jobs for AnnData files and derived SCP file fragments
+# usage patterns:
+# extract SCP files: AnnDataIngestParameters.new(anndata_file: study_file.gs_url)
+# parse extracted cluster file: AnnDataIngestParameters.new(
+#   ingest_anndata: false, extract_cluster: false, name: 'X_tsne', ingest_cluster: '--flag-only', domain_ranges: "{}"
```

Comment:
`--flag-only` seems unnecessarily unintuitive.  After reading, I understand it's used to construct a CLI argument in flag format -- i.e., only passing the name of the argument as an implied "true"; no argument value is passed.

I think it'd be a lot more understandable at a glance, and better encapsulated, to make such parameters in Ruby simple Booleans.  Using a specific string for effectively `true` while using a Boolean for `false` is unintuitive.  I can see why it was done this way, I just think it requires maintainers to build too much mental context.  

In other words, I think the benefit this particular mapping of the Ruby method interface to that Python command interface is outweighed by the impeded code understandability of mixing strings and Booleans for these parameter values.  It's too tightly coupled.  

All Boolean arguments in the Ingest Pipeline CLI have this form, so any argument that has `true` in Ruby bindings for the Ingest Pipeline CLI can be serialized as a flag-only argument.  Simple Booleans seems easier for maintainers to understand at a glance.

Reply:
That makes total sense - I'll figure out a way to simplify that interface.  Agreed it is non-intuitive.

Reply:
[c457c29](https://github.com/broadinstitute/single_cell_portal_core/pull/1689/commits/c457c29f9c13e6a4f6ff87723a3b5d44b778c0e7)